src/scripts/quests/QuestLineHelper.ts

```diff
@@ -1495,6 +1495,15 @@ class QuestLineHelper {
 
         App.game.quests.questLines().push(plasmaUnovaQuestLine);
     }
+    // XD Questline, available after Unova E4
+    public static createOrreXDQuestLine() {
+        const orreXDQuestLine = new QuestLine('Gale of Darkness', 'Team Cipher has returned to Orre. Stop their new evil plan!', new DevelopmentRequirement(), GameConstants.BulletinBoards.Hoenn);
```

Comment:
I think the questline should be in the unova board. Makes it easier for players to discover it.
Also, i think we want the players to complete the colosseum questline first, right?

Reply:
Yes to both. The quest line in this PR is just a placeholder now, next PR will incorporate both of those changes